twitter/statuses.go

```diff
@@ -55,6 +55,12 @@ func (t Tweet) CreatedAtTime() (time.Time, error) {
 	return time.Parse(time.RubyDate, t.CreatedAt)
 }
 
+// GetURL return the url of tweet
```

Comment:
please update the 3 return values

Reply:
Hi @davetweetlive,
Can you explain a little more? I don't get what you mean?
Thank you. 👍